### DISCUSSION TITLE
Select what Cpu Cores we want to use

### DISCUSSION THREAD

**Discussion by OzboiMr** [madMAx43v3r/chia-plotter#318] [Ideas]
Is there a way we can choose cpu cores ? on swar we were able to choose cpu cores like this : 

enable_cpu_affinity: false or true   cpu_affinity: [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23 ]

**Comment by SebMoore** [comment]
Use the linux command `taskset`
For example:
`taskset -c 1,2,3,4,5,6,7,8 ./chia-plotter [insert the options you want here as usual]`
would run chia-plotter with the options you specify on cores 1-8 of your system.

**Reply by SebMoore** [reply]
Just note that the second sentence of your question answers why this program will not include this option: you could do it on swar with the official plotter. This program is not a plot manager, it is the barebones plotter. See #60 for madMAx43v3r's stance on this.

**Reply by OzboiMr** [reply]
I have windows 😔
Is there a command for windows ?

**Reply by SebMoore** [reply]
Oh! Sure - the "start" command for windows includes a switch for CPU affinity - I haven't tried it myself, but here is a link that should generate some of the command for you - just scroll down a bit to see examples.
https://bitsum.com/tools/cpu-affinity-calculator/
